### DISCUSSION TITLE
Suggestion: 1.18.2 Wicker recipe change

### DISCUSSION THREAD

**Discussion by ZephyrWindSpirit** [malcolmriley/minecraft-textiles#63] [Ideas and Suggestions]
Both the sugar cane and fiber recipes produce 3 wicker patches each. In my own opinion, perhaps the flax fiber stalks could produce more? Like 4? It already takes more then two crafts of wicker patches to craft a basket. 

Are they placeable, these wicker patches?

**Comment by malcolmriley** [comment]
You may already know this, but recipes can be overridden with datapacks.

I left the wicker crafting yield at parity with the sugarcane recipe in the base mod because Flax matures more quickly than Sugarcane, and is more prolific (multiple stalks yielded per block of crop). In this way, the Flax to Wicker recipe already seemed "cheaper" and allows you to keep your Sugarcane stalk ... stock ... for paper and what-have-you. However I'm open to discussion on any balance points of course.

**Comment by malcolmriley** [comment]
The wicker patches are not placeable, they are just a crafting item.

**Comment by ZephyrWindSpirit** [comment]
Aw, the wicker isn't placeable. Thats fine.

I think sugar cane should offer 4 wicker then, since it does take longer. That way its not leaving a pile of extra wicker patches laying around.

**Comment by malcolmriley** [comment]
I actually do like the fact that there is a "good recipe" and a "not-so-good" recipe even if they have the same yield. Player choices and all that. I may be missing some nuance to your suggestion however.

**Comment by ZephyrWindSpirit** [comment]
Right now the wicker patch recipe for basket making reminds me of the previous versions of immersive engineering, where you had a single coke oven brick/blast furnace brick remaining after crafting the item. 
With what you've got at 3, you need 3 crafts of wicker patches to get your first basket, and 2 patches remaining. Two more crafts, make another basket, 1 patch remaining... you need a total of NINE craftings of wicker patches for breaking even on making baskets with patches with nothing leftover, if my math is correct?

**Reply by malcolmriley** [reply]
I think that was the part that I was overlooking. I've never found recipe leftovers to be bothersome so that wasn't on my mind, but I do get that they can feel bad.

I don't know if modifying the recipe yield is a better solution than just having more uses for the wicker patch (which is inevitable). There are many "recipe leftover" situations in the base game (like sticks in fences), but I suspect they aren't bothersome because the leftover items can be used for other things too.

I'll review the repository Project cards and see if I can't find something that could make sense to consume the wicker overage from basket-crafting.
